sdk/core/core/src/az_span.c

```diff
@@ -391,6 +391,45 @@ AZ_NODISCARD az_result az_span_append_int64(az_span * self, int64_t n) {
   return _az_span_builder_append_uint64(self, n);
 }
 
+static AZ_NODISCARD az_result _az_span_builder_append_uint32(az_span * self, uint32_t n) {
```

Comment:
Update to take az_span and return az_span *.
Also, update function to end with u32toa

Reply:
Got it. Changed.